src/events/http.js

```diff
@@ -215,6 +216,69 @@ function httpWrapper(wrappedFunction) {
                 this, buildParams(url, options, patchedCallback)
             );
 
+            /**
+             * Wraps 'on' method in a response to capture data event.
+             * @param {Function} wrappedResFunction The wrapped end function
+             * @returns {Function} The wrapped function
+             */
+            function responseOnWrapper(wrappedResFunction) { // eslint-disable-line no-inner-declarations,max-len
```

Comment:
must it be defined inside `httpWrapper`? it is getting very messy. better get `chunks` as a parameter (by adding another wrapping level) - not a must just my personal opinion, let me know if you think otherwise

Reply:
same with the next one

Reply:
these eslint disables share my opinion I guess

Reply:
👍